README.md

```diff
@@ -124,6 +124,9 @@ We'll get into all those classes in this document. The important thing to notice
 ### Dashboards
 - [Create a Dashboard](docs/dashboard.md)
 
+### Artisan Generators
+- [Artisan Generators](docs/artisan-generators.md)
```

Comment:
Maybe put this link in the "Generalities" section: creating a new "Artisan Generators" section seems to much since it will only contain one link...